app-frontend/src/app/components/pages/project/layer/scenes/scenes/index.js

```diff
@@ -29,6 +32,8 @@ class LayerScenesController {
             });
         this.fetchPage();
         this.setMapLayers();
+        this.closeUploadStatusCard = false;
```

Comment:
We generally want to use an adjective for variables instead of a verb.
`{object/target}{state}` vs `{action}{target/state}`
In this case: `uploadStatusCardClosed` instead of `closeUploadStatusCard`

The rational for this is that methods / functions do things, while variables just describe how things currently are.